Client/src/components/layout/Links/Nav/Nav.modules.css

```diff
@@ -0,0 +1,19 @@
+.Nav {
+  list-style: none;
+}
+
+.Nav a {
+  text-decoration: none;
+  padding: 10px 8px;
+  color: inherit;
+  margin: 0 4px;
+  border-radius: 4px;
+  transition: 180ms;
+}
+
+.dark .Nav a:hover {
+  background-color: rgb(68, 68, 68);
```

Comment:
TRy to alway avoid hardcoded colors. Create a palette and variables. It will be a nightmare later on when you need to perform rebranding and change colors alltogether... Applies everywhere

Reply:
@ivanshoshkov this is still unaddressed